src/server/verification/verificationAPI.js

```diff
@@ -188,7 +188,7 @@ const setup = (app: Router, verifier: VerificationAPI, storage: StorageAPI) => {
 
         if (verified === false) return
 
-        await storage.updateUser({ identifier: user.loggedInAs, smsValidated: true, mobile: savedMobile })
```

Comment:
use this savedMobile field in /user/add

Reply:
I take email and a mobile phone from the otp object, otp has this data and it is not overwritten